src/scenes/AcceleratorRouter/Participants/useNavigateToParticipants.tsx

```diff
@@ -0,0 +1,17 @@
+import { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { APP_PATHS } from 'src/constants';
+
+export type Response = () => void;
+
+export default function useNavigateToParticipants(): Response {
```

Comment:
This is nice, maybe moving forward we should wrap all the relevant routes for an area in a hook like this so we can easily sprinkle them around

Reply:
That's a nice idea, we could have a library of navigation hooks.. same for table cell renderers. I will create a tech investment ticket.

Reply:
Moved the useNavigateTo.. under `src/hooks/navigation` so we can add to that directory.

Reply:
Created https://terraformation.atlassian.net/browse/SW-5104